amundsen_application/static/js/interfaces/Resources.ts

```diff
@@ -38,6 +38,7 @@ export interface TableResource extends Resource {
   last_updated_timestamp?: number;
   name: string;
   schema: string;
+  schema_description: string;
```

Comment:
This should be an optional value using ` schema_description?: string;`. As you may have noticed this value only exists on the the search results, and there are other parts of the application where the `TableListItem` shows up where we won't be showing this. Making this optional will also prevent you from having to have added this into all the other files that aren't particularly related to this feature.

Reply:
okay now I understand why I have to make changes into other files. Thanks!